### PR TITLE
feat: Added My cart

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

**Review by rahulk31 — APPROVED** [review]
UX suggestion: There should not be login button on My cart Page.

image


UI Suggestion: These division lines can be dimmed in color. right now it looks harsh.
image

Overall code looks good to me. 🚀

**Comment by Ankur9669** [comment]
> UX suggestion: There should not be login button on My cart Page.
> 
> image
> 
> UI Suggestion: These division lines can be dimmed in color. right now it looks harsh. image
> 
> Overall code looks good to me. 🚀

Thanks for your suggestions bro, i would remove the login button from here, but the division lines looks fine to me, i would definately ask other users what they feel and if they fell it is harsh, i would dim it😀

**Comment by rahulk31** [comment]
> 

Sure Ankur

**Comment by Naman-Saxena1** [comment]
Looks good to me 👍 
And about suggestion of dimming line by Rahul, you can do it but it not a major issue.
You can also use 1 more color if you want to grab user's attention (Eg. Towards Buy Now Button), right now all buttons on screen have same color. But it is also fine if you want to use only 1 color in your theme, up to you.

File: mycart.css [hunk 1]
```diff
@@ -1,19 +1,101 @@
...
 }
 .mycart-summary-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
     grid-column: 2 / 3;
+}
+.my-cart-card{
+    /* This padding is to override the css coming from CL */
+    padding: 0rem 0rem;
```
Comment (nikhiljugale007) [inline mycart.css:20]: Can we use padding: 0 here instead of padding : 0 0;

**Review by nikhiljugale007 — COMMENTED** [review]
One suggestion/doubt in code else good to merge as per me.

File: mycart.css (hunk 1 above)
Comment (Ankur9669) [reply mycart.css:20]: Nope that is just to override the css coming to CL😂

**Comment by Ankur9669** [comment]
> Looks good to me 👍 And about suggestion of dimming line by Rahul, you can do it but it not a major issue. You can also use 1 more color if you want to grab user's attention (Eg. Towards Buy Now Button), right now all buttons on screen have same color. But it is also fine if you want to use only 1 color in your theme, up to you.

Ya that is a nice suggestion but currently i am thinking to use only 1 cta color for buttons, in future i might change its color, i would note it for now.

**Pull request merged**